### PR TITLE
fix: 🐛  fix timer skipping and added timer for prompt

### PR DESCRIPTION
**Pull request by gchamb**

### CODE REVIEW

**Comment by nstringham** [comment]
related to #87

File: client/src/components/Prompt.svelte [hunk 1]
```diff
@@ -39,6 +65,9 @@
   }
 </script>
 
+{#if countdown != undefined}
+  <h1 class="mdc-typography--headline3" style="text-align:center ;">{countdown}</h1>
```
Comment (nstringham) [inline client/src/components/Prompt.svelte]: can we use the `mm:ss` thing?

**Review by nstringham — DISMISSED** [review]

File: client/src/components/Prompt.svelte [hunk 2]
```diff
@@ -4,20 +4,47 @@
   import HelperText from "@smui/textfield/helper-text";
   import { getPromptAnswerCollection } from "$lib/firebase/firestore-collections";
   import { doc, setDoc } from "firebase/firestore";
+  import { GAME_STATE_DURATIONS, type Lobby } from "$lib/firebase/firestore-types/lobby";
+  import { authStore as user } from "$stores/auth";
+  import { verifyExpiration } from "$lib/firebase/firebase-functions";
+    import { formatTimer } from "$lib/time";
```
Comment (njumaah) [inline client/src/components/Prompt.svelte]: remove tab

**Review by njumaah — APPROVED** [review]
lgtm

**Pull request merged**